### ISSUE TITLE
Add support for item type movie

### ISSUE THREAD

**Issue by Gettyk** [jlegewie/zotfile#270]
If I have a file under an item with a movei type, Zotfile doens't use the name in contributor, director, scriptwriter or producer in the renaming rule (%a). 

That would be great to have this feature!

**Comment by jlegewie** [comment]
Yes, makes sense. In the meantime, you can use user-defined wildcards, which should make this possible. Please post on the Zotero forum if you run into problems with wildcards.
